Renaming salt-ssh option and documentation fix

MorphBonehunter opened the pull request:
For consistency the wipe_ssh option is renamed in ssh_wipe as all other
options in the salt-ssh command are in this style.
The documentation was also adjusted to reflect the renaming.
There were also an wrong salt-ssh example in the documentation.

MorphBonehunter committed:
Renaming salt-ssh option and documentation fix

For consistency the wipe_ssh option is renamed in ssh_wipe as all other
options in the salt-ssh command are in this style.
The documentation was also adjusted to reflect the renaming.
There were also an wrong salt-ssh example in the documentation.

rallytime commented:
Go Go Jenkins!

basepi commented:
@MorphBonehunter This will break existing setups. Could you please look for either version of the config option and if they're using the old one, throw a warning using `warn_until`? If you grep the codebase for `warn_until` you'll see many examples of how this is done. We need to give people a deprecation path.

MorphBonehunter committed:
Add backward compatibility for wipe_ssh

Added an check for the deprecated config option wipe_ssh and print
an warning about the deprecation.

MorphBonehunter commented:
@basepi Ok, i understand this.
Only problem, i'm not an developer/programmer (to be honest, i do not really understand any programming language) and so i do not know if the push i've done a few seconds ago is valid or at least functional. Maybe you can check if i did the right thing.

basepi commented:
You did perfectly. I think we should target Nitrogen, rather than Carbon, because that way they'll have two releases to notice the warnings. Would you mind pushing one more commit changing "Carbon" to "Nitrogen"? Thanks so much!

MorphBonehunter committed:
Extend deprecation warning until Nitrogen

Show the wipe_ssh deprecation warning until Nitrogen is released.

MorphBonehunter commented:
Thanks for your feedback!
I've changed the Version from Carbon to Nitrogen.

basepi commented:
Perfect!
